@kiva/kv-stream/src/index.ts

```diff
@@ -0,0 +1,55 @@
+import {
+	connect,
+	StreamClient,
+	GetFeedOptions,
+	NewActivity,
+	StreamFeed,
+	FeedAPIResponse,
+	Activity,
+	APIResponse,
+} from 'getstream';
+
+export class StreamService {
```

Comment:
Similarly, `ActivityFeedService` might be a good abstraction, since "stream" has different potential meanings.